src/style.css

```diff
@@ -2,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Book';
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url(/src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 800;
+}
+
+body {
+  margin: 0 10px;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.bar {
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  align-items: center;
+
+  display: inline-block;
+}
+
+.big-bar {
+  font-family: Avenir, sans-serif;
+  background-position: 26px 25px;
+  background-size: 19px;
+
+  padding-left: 62px;
+  margin: 20px 0;
```

Comment:
Ensure that the margin between the search bars is set to 20px as per the design specifications. This line seems to address that requirement, but please verify its effect in the layout.